src/GafferSceneUI/LightPositionTool.cpp

```diff
@@ -669,16 +671,18 @@ void LightPositionTool::updateHandles( float rasterScale )
 	if( !m_drag )
 	{
 		bool singleSelection = selection().size() == 1;
+		bool isLight = s.scene()->set( g_lightsSetName )->readable().match( s.path() ) == IECore::PathMatcher::ExactMatch;
 
 		TranslationRotation trDistanceHandle( s, Orientation::World );
 		m_distanceHandle->setEnabled(
 			singleSelection &&
+			isLight &&
 			trDistanceHandle.canApplyTranslation() &&
 			trDistanceHandle.canApplyRotation( V3i( 1, 1, 1 ) )
 		);
 
 		TranslationRotation trRotateHandle( s, Orientation::Local );
-		m_rotateHandle->setEnabled( singleSelection && trRotateHandle.canApplyRotation( V3i( 0, 0, 1 ) ) );
+		m_rotateHandle->setEnabled( singleSelection && isLight && trRotateHandle.canApplyRotation( V3i( 0, 0, 1 ) ) );
```

Comment:
I'm seeing a disabled rotate handle at the origin when I have a non-light selected. Can we hide it instead in this case?

![image](https://github.com/GafferHQ/gaffer/assets/1133871/9b034266-f3ee-4499-b146-7ce99ee89fde)

Reply:
Fixed and squashed into https://github.com/GafferHQ/gaffer/pull/5723/commits/47d14183b7c818a0cc9b864110ace36d8bc88955